### ISSUE TITLE
Fix links on track-update page

### ISSUE THREAD

**Issue by joshgoebel** [exercism/exercism#5868]
<img width="985" alt="Screen Shot 2021-09-10 at 7 34 07 PM" src="https://user-images.githubusercontent.com/6473/132927794-ee2cfd23-399e-4927-886a-fca313675da8.png">

Every item has a more / details / disclosure / arrow whatever you want to call it, but only the top one is clickable. What is the arrow signifying if I can't get more details.  I think the arrows should be removed, save the first for clickable item.

**Comment by iHiD** [comment]
They should take you to the various iterations.

**Comment by joshgoebel** [comment]
They are not clickable nor for they have a clickable cursor indicator (Safari and Chrome).

**Comment by joshgoebel** [comment]
This in on main track page: https://exercism.org/tracks/wren

**Comment by angelikatyborska** [comment]
@iHiD They're just divs, not links:

![Screen Shot 2021-09-12 at 21 16 41](https://user-images.githubusercontent.com/7852553/132999992-07c3a29d-a2b5-4669-a94d-0ee028b3ead0.png)

**Comment by iHiD** [comment]
Yeah, to be clear, I'm disputing it's broken, just answering the question of why it's this way. It's just unfinished atm. I'll reword the title to indicate it needs fixing.
